api/app/controllers/spree/api/option_types_controller.rb

```diff
@@ -3,9 +3,9 @@ module Api
     class OptionTypesController < Spree::Api::BaseController
       def index
         if params[:ids]
-          @option_types = Spree::OptionType.accessible_by(current_ability, :read).where(:id => params[:ids].split(','))
+          @option_types = Spree::OptionType.includes(:option_values).accessible_by(current_ability, :read).where(id: params[:ids].split(','))
```

Comment:
Line is too long. [141/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.